dashboard/app/models/user_school_info.rb

```diff
@@ -13,6 +13,8 @@
 #
 
 class UserSchoolInfo < ApplicationRecord
-  validates_presence_of :user
-  validates_presence_of :school_info
+  validates_presence_of :user, :school_info, :start_date, :last_confirmation_date
+
+  belongs_to :user
```

Comment:
These changes are in PR #27035 so there might be merge conflicts when that one is merged.

Reply:
@bethanyaconnor Thanks for the heads up.  I will resolve the conflict - last confirmation date & start date validations are not in PR 27035 so those will only be the updates.